skimage/segmentation/_clear_border.py

```diff
@@ -16,7 +16,8 @@ def clear_border(labels, buffer_size=0, bgval=0, in_place=False):
         Cleared objects are set to this value.
     in_place : bool, optional
         Whether or not to manipulate the labels array in-place.
-
+    mask : ndarray of bool, same shape as `image`, optional.
+        Image data mask.
```

Comment:
Could you explain a bit more what ``mask`` is doing? Like "objects containing True pixels of ``mask`` will be removed".